space_packet_parser/packets.py

```diff
@@ -2,34 +2,46 @@
 """Packet containers and parsing utilities for space packets."""
 
 from dataclasses import dataclass, field
-from typing import Union, Optional, Protocol, List
+from typing import Any, Optional, Protocol, List
 
 
-@dataclass
-class ParsedDataItem:
-    """Representation of a parsed parameter
+class DataItem:
```

Comment:
Gah I really want to call this `Parameter`, but alas...
Not for this PR but wondering if we should prefix all the XTCE UML model classes with `Xtce`.

Reply:
If you agree with #88 , you can incorporate it here, or not.

Reply:
I made this a private `_Parameter` as it really shouldn't be used by other people, it is for internal use.

I updated everything to `StrParameter` for the naming now. Is that what you had in mind? I did not modify any of the other "Parameter" entries.

It is a value here though and not a "parameter" right? So do we want to add or include "Value" or "Data" along with Parameter?

Reply:
I think semantically most people will assume a Parameter means a value but I'd also be happy with StrParameterValue if you think that's clearer.